apps/data-types.md

```diff
@@ -52,26 +52,81 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | minCartValue | object of [price](page:apps-data-types#price) | The minimum order value of a shop. |
 | checkoutURL | string | The URL that redirects the browser to the merchant’s shop in order to complete the checkout. |
 
+## cartIdAddressPutRequest
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| firstName | string | The first name of the person associated with the address.|
+| lastName | string | The last name of the person associated with the address.|
+| street | string | The street name of the address. |
+| zipCode | string | The zip or postal code of the address. |
+| city | string | The name of the city. |
+| state | string | The name of the state.|
+| country | string | The name of the country. |
+| birthday | string | The date of birth of the person associated with the address. |
+| emailAddress | string | The email address of the person associated with the address. |
+
+## cartPostRequest
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| currency | string | The currency code of the price according to ISO 4217.|
+| taxType | string | Indicates if the amount includes tax, e.g. gross.|
+| locale | string | The locale that identifies the origin of the customer.|
+| lineItems | array of [cartIdLineItemPostRequest](page:apps-data-types#cartidlineitempostrequest)  | The product line items in the basket.|
+
+## cartIdLineItemPostRequest
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| productId | string | The unique identifier of the product.|
+| quantity | string | The quantity of the product line item.|
```

Comment:
@ubep for amount the type is *number*, isn't it inconsistent to use decimal here? Or is there definitely a difference?